crates/holochain_conductor_config/src/ports.rs

```diff
@@ -21,9 +21,14 @@ pub fn set_admin_port(config: &mut ConductorConfig, port: u16) {
         .and_then(|ai| ai.get_mut(0))
     {
         Some(admin_interface) => {
-            *admin_interface = port;
```

Comment:
This was overriding the `allowed_origins` field as well which made it impossible to write a test because the modified config would always be set back to the default value here.